src/main/scala/is/hail/sparkextras/OrderedRDD2.scala

```diff
@@ -759,6 +759,30 @@ class OrderedRDD2 private(
     rdd: RDD[RegionValue] = rdd): OrderedRDD2 = {
     OrderedRDD2(typ, orderedPartitioner, rdd)
   }
+
+  def naiveCoalesce(maxPartitions: Int): OrderedRDD2 = {
+    val n = orderedPartitioner.numPartitions
+    if (maxPartitions > n)
```

Comment:
This should be `>=`, and the documentation of the `max_partition parameter` should say "less than or equal" rather than "less than".